deploy/helm/sumologic/templates/_helpers.tpl

```diff
@@ -412,15 +412,11 @@ helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
 {{- end -}}
 
 {{- define "sumologic.metadata.name.logs" -}}
-{{- if eq .Values.sumologic.logs.metadata.provider "fluentd" -}}
-{{ template "sumologic.metadata.name.fluentd" . }}-logs
-{{- else if eq .Values.sumologic.logs.metadata.provider "otelcol" -}}
 {{ template "sumologic.metadata.name.otelcol" . }}-logs
```

Comment:
We should leave statefulset names as they are. AFAIK, After this change we can end with fluentd statefulset named otelcol

Reply:
what I was trying to do here was to use `otelcol` regardless of  the provider.

Reply:
actually, I see what you mean, we can't change the name of a StatefulSet on upgrade. I'll change the default provider for logs and metrics to otelcol.